packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -580,6 +586,11 @@ export class GarbageCollector implements IGarbageCollector {
             this.sweepTimeoutMs =
                 testOverrideSweepTimeoutMs
                 ?? computeSweepTimeout(this.sessionExpiryTimeoutMs);
+            if (this.gcOptions[gcTombstoneGenerationOptionName] !== undefined) {
+                this.persistedGcFeatureMatrix = {
+                    tombstoneGeneration: this.gcOptions[gcTombstoneGenerationOptionName],
```

Comment:
Are we not setting whether or not tombstone is enabled?

Reply:
Not in this PR. This just lets Loop start writing the value to a file to get the 36-day clock ticking until an object would even become Tombstoned. Otherwise, if my main change isn't integrated until end of Feb, then it's not until April they start getting real feedback.